src/index.html

```diff
@@ -20,8 +21,523 @@
       href="styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="header__logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="header__logo"
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-list-item">
+            <a
+              href="#Apple"
+              class="header__link header__link--is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__nav-list-item">
+            <a
+              href="#Samsung"
+              class="header__link"
+            >
```

Comment:
Fix code style everywhere if you have only 2 attributes
```suggestion
            <a href="#samsung" class="nav__link">
```